zkevm-circuits/src/util/word.rs

```diff
@@ -323,6 +324,18 @@ impl<F: Field> From<H160> for Word<F> {
     }
 }
 
+impl<F: Field> From<eth_types::Hash> for Word<F> {
+    /// Construct the word from eth_types::Hash
+    fn from(value: eth_types::Hash) -> Self {
+        let mut bytes = *value.as_fixed_bytes();
+        bytes.reverse();
+        Word::new([
+            from_bytes::value(&bytes[..N_BYTES_HALF_WORD]),
+            from_bytes::value(&bytes[N_BYTES_HALF_WORD..]),
+        ])
+    }
+}
+
```

Comment:
We already have `From<eth_types::Word>` in line 283.
```suggestion
```

Reply:
In this branch there is not this in line 283, I think

Reply:
ok, there are from eth_types::Word and from eth_types::Hash. One alias U256 and other H256, is there a plan to don't use one of them? I am not sure why we don't need both?